use-cases/using-SPDK-vhostuser-and-kata.md

```diff
@@ -0,0 +1,120 @@
+# Setup to run SPDK vhost-user devices with Kata Containers and Docker*
+
+- [SPDK Vhost-user target overview](#spdk-vhost-user-target-overview)
+- [Install and setup SPDK vhost-user target](#install-and-setup-spdk-vhost-user-target)
+- [Host setup for vhost-user devices](#host-setup-for-vhost-user-devices)
+- [Launch a Kata container with SPDK vhost-user-blk device](#launch-a-kata-container-with-spdk-vhost-user-blk-device)
+
+## SPDK Vhost-user Target Overview
+
+The Storage Performance Development Kit (SPDK) provides a set of tools and libraries for writing high performance, scalable, user-mode storage applications.
+
+While, virtio is a efficient way to transport data for virtual environments and guests. It is most commonly used in QEMU VMs, where the QEMU itself exposes a virtual PCI device and the guest OS communicates with it using a specific Virtio PCI driver.
+Vhost is a protocol for devices accessible via inter-process communication. It uses the same virtqueue layout as Virtio to allow vhost devices to be mapped directly to virtio devices. The initial vhost implementation is a part of the Linux kernel and uses ioctl interface to communicate with userspace applications.
+Vhost-user implements the control plane through Unix domain socket to establish virtqueue sharing with a user space process on the same host. So SPDK can expose vhost device via Vhost-user protocol.
+
+SPDK vhost is a Vhost-user slave server. It exposes Unix domain sockets and allows external applications to connect. It is capable of exposing virtualized storage devices to QEMU instances or other arbitrary processes.
+
+Currently, SPDK vhost-user target can exposes these types of virtualized devices:
+
+- vhost-blk
+- vhost-scsi
+- vhost-nvme
+
+For more information, visit [SPDK](https://spdk.io) and [SPDK vhost-user target](https://spdk.io/doc/vhost.htm).
+
+## Install and setup SPDK vhost-user application
+
+### Get source code and build SPDK
```

Comment:
could you include these subsections in the index table under `Install and setup SPDK vhost-user application` section?